demo/element-basic-demos.html

```diff
@@ -10,6 +10,24 @@ <h3>Basics</h3>
     <vaadin-demo-snippet id="element-basic-demos-sample-example">
       <template preserve-content>
         <vaadin-login></vaadin-login>
+        <vaadin-dialog id="login">
+          <template>Login is being processed...</template>
+        </vaadin-dialog>
+        <script>
+          window.addDemoReadyListener('#element-basic-demos-sample-example', function(document) {
+            var login = document.querySelector('vaadin-login');
+            var loginDialog = document.querySelector('#login');
```

Comment:
Varibales and element id look a bit confusing from the first sight

Reply:
Changed to `feedbackDialog` for both dialog id and variable name.